src/mono/mono/mini/aot-compiler.c

```diff
@@ -3057,7 +3057,7 @@ stream_init (MonoDynamicStream *sh)
 }
 
 static void
-make_room_in_stream (MonoDynamicStream *stream, int size)
+make_room_in_stream (MonoDynamicStream *stream, guint32 size)
```

Comment:
Let's use C99 types instead of glib types

```suggestion
make_room_in_stream (MonoDynamicStream *stream, uint32_t size)
```